src/core/bios.cpp

```diff
@@ -47,9 +47,14 @@ static constexpr std::array<ImageInfo, 21> s_image_infos = {{
    MakeHashFromString("417b34706319da7cf001e76e40136c23")},
   {"SCPH-1002, DTL-H1202, H3002 (v2.2 12-04-95 E)", ConsoleRegion::PAL,
    MakeHashFromString("e2110b8a2b97a8e0b857a45d32f7e187")},
+  {"SCPH-1002, DTL-H1100 (Version 2.2 03/06/96 D)", ConsoleRegion::PAL,
+   MakeHashFromString("ca5cfc321f916756e3f0effbfaeba13b")},
```

Comment:
Are you sure this is a PAL region BIOS? From what I'm aware, this is an NTSC-J BIOS exclusive to the DTL-H1100, hence ending on a 0 and having a D suffix.